code/modules/research/designs/misc_designs.dm

```diff
@@ -284,7 +295,26 @@
 	materials = list(/datum/material/iron=200, /datum/material/glass = 100)
 	build_path = /obj/item/card/id
 	category = list("Electronics")
-	departmental_flags = DEPARTMENTAL_FLAG_SERVICE	
+	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
+
+/datum/design/eng_gloves

```

Comment:
There are 2 designs not in the PR description here.

Reply:
What?

Reply:
These seem to be these BEPIS designs from #48507

Reply:
Those are old bepis designs that I copy pastaed instead of updating my git because I'm lazy as fuck

Reply:
Git hub was mad that I replaced an empty line with a tab because there was already a new design there.

Reply:
That code is already in the TG master

Reply:
wack